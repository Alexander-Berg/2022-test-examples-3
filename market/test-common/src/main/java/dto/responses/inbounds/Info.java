package dto.responses.inbounds;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class Info {

    @JsonProperty("boxes")
    private List<Object> boxes;

    @JsonProperty("unsortedPallets")
    private List<Object> unsortedPallets;

    @JsonProperty("unpackedBoxes")
    private List<Object> unpackedBoxes;

    @JsonProperty("pallets")
    private List<Object> pallets;

    public List<Object> getBoxes() {
        return boxes;
    }

    public List<Object> getUnsortedPallets() {
        return unsortedPallets;
    }

    public List<Object> getUnpackedBoxes() {
        return unpackedBoxes;
    }

    public List<Object> getPallets() {
        return pallets;
    }
}
