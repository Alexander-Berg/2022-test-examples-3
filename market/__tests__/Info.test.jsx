import React from 'react';
import { create } from 'react-test-renderer';

import Info from '../index';

describe('Info', () => {
    test('model found', () => {
        const price = { value: 75000, currencyName: 'руб.' };
        const c = create(<Info data={() => ({ price })} />);
        expect(c.toJSON()).toMatchSnapshot();
    });

    test('model not found', () => {
        const price = { offersCount: 10 };
        const c = create(<Info data={() => ({ price })} />);
        expect(c.toJSON()).toMatchSnapshot();
    });
});
