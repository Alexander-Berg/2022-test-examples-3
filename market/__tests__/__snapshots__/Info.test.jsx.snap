// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info model found 1`] = `
<div
  className="info"
>
  <div
    className="prices"
  >
     
  </div>
  <img
    alt=""
    className="arrow"
    src={Object {}}
  />
</div>
`;

exports[`Info model not found 1`] = `
<div
  className="info"
>
  <div
    className="prices"
  >
     
  </div>
  <img
    alt=""
    className="arrow"
    src={Object {}}
  />
</div>
`;
